tests/lax_scipy_special_functions_test.py

```diff
@@ -317,6 +317,31 @@ def testGammaIncCBoundaryValues(self):
     self._CheckAgainstNumpy(lsp_special.gammaincc, osp_special.gammaincc, args_maker, rtol=rtol)
     self._CompileAndCheck(lsp_special.gammaincc, args_maker, rtol=rtol)
 
+  def testBetaIncBoundaryValues(self):
+    dtype = jax.numpy.zeros(0).dtype  # default float dtype.
```

Comment:
More direct is `jax.dtypes.canonicalize_dtype(float)`

Reply:
Applied, thanks!